packages/components/src/ui/heading.tsx

```diff
@@ -45,7 +45,6 @@ export function EditableH1(props: {
   return (
     <div>
       <h1
-        // eslint-disable-next-line jsx-a11y/no-noninteractive-element-to-interactive-role -- messy fix should be reworked
```

Comment:
This is still needed, the only reason this wasn't throwing a warning was because the wrong eslint config was used